systemtests/battery_test.go

```diff
@@ -28,150 +28,181 @@ func (s *systemtestSuite) TestBatteryMultiMountSameHost(c *C) {
 		}()
 	}
 
+	var realErr error
+
 	for x := 0; x < count; x++ {
-		c.Assert(<-errChan, NotNil)
+		err := <-errChan
+		if err != nil {
+			realErr = err
```

Comment:
I might be reading this wrong. But I think we should `break` on first `non-nil` error, otherwise `realErr` might get overwritten when a command that finishes last returns a nil error